src/Message/PurchaseResponse.php

```diff
@@ -1,113 +1,108 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Omnipay\SchoolEasyPay\Message;
 
 use Omnipay\Common\Message\AbstractResponse;
+use Symfony\Component\HttpFoundation\Response;
 
 /**
  * Response class for all SchoolEasyPay requests
  */
 class PurchaseResponse extends AbstractResponse
 {
-    /** @var string Request ID */
-    protected $requestId = null;
-    /** @var string HTTP response code */
-    protected $httpResponseCode = null;
+    protected string $requestId;
+
+    protected string $httpResponseCode;
 
     /**
      * Is the transaction successful?
-     * @return boolean True if successful
      */
-    public function isSuccessful()
+    public function isSuccessful(): bool
     {
         // get response code
         $code = $this->getHttpResponseCode();
```

Comment:
Same question here regarding this being string rather than int.

Reply:
Added comment here https://github.com/digistorm/omnipay-windcave/pull/5#discussion_r1843043599